includes/RemovePIIJob.php

```diff
@@ -434,7 +435,7 @@ public function run() {
 		// Hide deletions from RecentChanges
 		$userGroupManager->addUserToGroup( $user, 'bot', null, true );
 
-		$dbr = wfGetDB( DB_REPLICA, [], $this->database );
+		$dbr = wfGetDB( DB_REPLICA );
```

Comment:
here as well, although if you're just doing a few queries off the replica while doing a bunch of writes might be simpler to just query from the primary instead of open a new connection to a random replica

Reply:
Done